### PR TITLE
added button to search

### PR DESCRIPTION
**Pull request by diannasb67**
Added button beneath the search bar on home and destination page

### CODE REVIEW

**Review by dibon001 — COMMENTED** [review]
adding a submit button

**Review by dibon001 — APPROVED** [review]
adding a submit button

File: assets/css/style.css [hunk 1]
```diff
@@ -19,11 +19,11 @@ p { padding-top: 10px;
    } */
 
 .factBlock {
-    height: 350px;
```
Comment (dibon001) [inline assets/css/style.css:22]: update height

**Review by dibon001 — APPROVED** [review]

File: assets/css/style.css [hunk 2]
```diff
@@ -19,11 +19,11 @@ p { padding-top: 10px;
...
+    height: 320px;
     
 }
 .facts {
-    height: 425px;
```
Comment (dibon001) [inline assets/css/style.css:26]: delete height 425

File: index.html [hunk 3]
```diff
@@ -89,7 +89,8 @@
       <h1 class="discover">Discover Your World</h1>
       <div class="form-group searchBar">Search a City
         <label for="formGroupExampleInput"></label>
-        <input type="text" class="form-control" id="formGroupExampleInput" placeholder="Search">
```
Comment (dibon001) [inline index.html:92]: added submit button

**Review by dibon001 — APPROVED** [review]

**Review by dibon001 — APPROVED** [review]

**Review by danielhomann4 — APPROVED** [review]

**Pull request merged**